### PR TITLE
API bug fixes for #1517

### PR DESCRIPTION
**Pull request by doctyper**
Small patch to fix API/CLI issues introduced in Modernizr/Modernizr#1517

### CODE REVIEW

File: feature-detects/ambientlight.js [hunk 1]
```diff
@@ -12,5 +12,5 @@
 Detects support for the API that provides information about the ambient light levels, as detected by the device's light detector, in terms of lux units.
 */
 define(['Modernizr', 'hasEvent'], function( Modernizr, hasEvent ) {
-  Modernizr.addTest('devicelight', hasEvent('devicelight', window));
```
Comment (doctyper) [inline feature-detects/ambientlight.js]: My Customizr tool fails testing due to this unreferenced test name. I could instead change the file name and leave the test name as it was. Open to either.

Comment (patrickkettner) [inline feature-detects/ambientlight.js]: `ambientlight` makes more sense to me

**Comment by patrickkettner** [comment]
great call on the package.json!

how is the cli dependant on `file`?

**Comment by doctyper** [comment]
It's used by the `metadata` endpoint [here](https://github.com/Modernizr/Modernizr/blob/master/lib/metadata.js#L2).

**Comment by patrickkettner** [comment]
doi - don't know how I missed that.

cheers

**Pull request merged**